backend/user_management/ai/llm.py

```
@@ -0,0 +1,46 @@
+    # If the user is searching for a t-shirt
+    if "t-shirt" in user_message or "tshirt" in user_message or "shirt" in user_message:
+        return get_matching_products("t-shirt")
+
+    # Detect "Select {product_id} {size}" command for ordering
+    if user_message.startswith("select"):
+        try:
+            parts = user_message.split(" ")
+            product_id = int(parts[1])
+            size = parts[2] if len(parts) > 2 else None  # Extract size if provided
+            return place_order(user_id, product_id, size)
+        except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)